src/Controls/tests/DeviceTests/Memory/MemoryTests.cs

```diff
@@ -435,6 +435,37 @@ public async Task WindowDoesNotLeak()
 		await AssertionExtensions.WaitForGC(references.ToArray());
 	}
 
+	[Fact("VisualDiagnosticsOverlay Does Not Leak"
+#if IOS || MACCATALYST
+		, Skip = "Fails with 'MauiContext should have been set on parent.'"
+#endif
+	)]
```

Comment:
I did not figure out how to get this test to work on iOS/Catalyst. It seems like it wants the Window's Handler to be created for it to work.